pandas/tests/extension/test_numpy.py

```diff
@@ -151,6 +151,16 @@ def test_array_from_scalars(self, data):
         # ValueError: PandasArray must be 1-dimensional.
         super().test_array_from_scalars(data)
 
+    @skip_nested
+    def test_series_constructor_scalar_with_index(self, data):
+        # ValueError: Length of passed values is 1, index implies 3.
```

Comment:
for the object dtype, the scalar is a tuple, so this failure is related to https://github.com/pandas-dev/pandas/pull/33846#discussion_r417937451